package.xml

```diff
@@ -18,9 +18,9 @@
   <buildtool_depend>ament_cmake_python</buildtool_depend>
   <buildtool_depend>python_cmake_module</buildtool_depend>
 
-  <build_depend>builtin_interfaces</build_depend>
-  <build_depend>rclcpp</build_depend>
-  <build_depend>rclpy</build_depend>
+  <depend>builtin_interfaces</depend>
+  <depend>rclcpp</depend>
+  <depend>rclpy</depend>
```

Comment:
Similar to `builtin_interfaces`, `rclpy` is only ever used in the Python code, so it should be an `exec_depend`.

Reply:
Indeed.